components/layout_2020/display_list/mod.rs

```diff
@@ -212,6 +212,11 @@ impl Fragment {
 
         let color = fragment.parent_style.clone_color();
         let font_metrics = &fragment.font_metrics;
+        let dppx = builder.context.style_context.device_pixel_ratio().get();
+        let adjust_thickness = |thickness: Length| -> Length {
```

Comment:
I think it might be better to call this `round_to_nearest_device_pixel` here and above.

Reply:
Done, also renamed the parameter to a more generic `value`.